pkg/utils/shared_objects/container_symbols_loader.go

```diff
@@ -0,0 +1,46 @@
+package shared_objects
+
+import (
+	"github.com/aquasecurity/tracee/pkg/containers"
+)
+
+// ContainersPathSOSymsLoaderDecorator is a decorator for SO loaders that resolves containers-relative paths to
+// absolute host paths.
+type ContainersPathSOSymsLoaderDecorator struct {
```

Comment:
The fact that this is a decorator seems irrelevant in name to what looks to be the intended usage (can't tell since there's no consuming code). Intent is the most important, let's keep it simple, something like `ContainerSymbolsLoader` as opposed to the `HostSymbolLoader`.

If later down the line this decorator is needed elsewhere, it should be split out and then given the decorator name, but currently this is just another `SODynamicSymbolsLoader` implementation, let's name it as such.